index.html

```diff
@@ -1,45 +1,132 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Marke Parker</title>
-    <link rel="stylesheet" href="./assets/css/style.css">
-</head>
-<body>
-<!-- This a Header Section [Moussa Ibrahim]-->
-<main>
-    <header>
-        <nav class="header__nav">
+    <link
+    rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Roboto&family=Rubik:wght@300&display=swap"
+      
+    />
+    <!-- googlefont -->
+    <link rel="stylesheet" href="./assets/css/font-awesome.css" />
+    <!-- fontawesome -->
+    <link rel="stylesheet" href="./assets/css/style.css" />
+  </head>
+  <body>
+    <!-- This a Header Section [Moussa Ibrahim]-->
+    <main>
+      <header>
+        <nav class="header__nav"></nav>
+      </header>
+    </main>
+    <!-- Read About My Life Section [SARA MAGDY]-->
+    <section class="about section--py section">
+      <div class="container about__container">
+        <div class="about__imgDiv">
+          <a href="#"><button class="about__btn">get in touch</button></a>
+        </div>
 
-        </nav>
-    </header>
-</main>
-<!-- Read About My Life Section [SARA MAGDY]-->
-<section class="about section--py section"></section>
+        <!-- End about__img -->
 
-<!-- My Work Process Section [Mohamed Saeed Saafan]-->
-<section class="work-process section--py section"></section>
+        <div class="about__content">
+          <h2 class="about__title">
+            read about my life <br />
+            struggle story
+          </h2>
+          <p class="about__desc">
+            Donec commodo augue vitae metus interdum molestie. Fusce dapibus non
+            mauris sit amet blandit. Maecenas quis massa nec sem laoreet
+            vehicula quis sit amet tortor. Proin ornare, lorem ut pellentesque
+            maximus, ex leo viverra lorem, ac dapibus neque tellus id dolor.
+            Curabitur luctus tortor eget dolor ultrices fermentum.
+          </p>
+          <!-- progress div -->
+          <div class="about__progress">
+            <div class="about__item about__item--development">
+              <h3 class="about__subtitle">
```

Comment:
why do u use heading for it ?

Reply:
What is the problem in using heading ? 😅
generally i used it to make all element block and it include the icon and span .